src/android/ApptentiveBridge.kt

```diff
@@ -16,11 +18,13 @@ import org.apache.cordova.CordovaPlugin
 import org.apache.cordova.PluginResult
 import org.json.JSONArray
 import org.json.JSONException
+import java.lang.ref.WeakReference
 
 @OptIn(InternalUseOnly::class)
 class ApptentiveBridge : CordovaPlugin(), ApptentiveActivityInfo {
 
   private var isApptentiveRegistered = false
+  private var savedCallbackContext: WeakReference<CallbackContext>? = null
```

Comment:
I don't know it's worth making `savedCallbackContext` to `null` in `onDestroy`. But wouldn't hurt.